### PR TITLE
chore: Refactor the background endpoint implementation

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
I would like to keep the approach where `seconds` argument is passed to this endpoint like in Android to avoid unnecessary confusion for different clients.

The final rules will be:
- seconds value can be float
- if seconds value is greater or equal to zero then the app is minimized and restored after the timeout
- if seconds less than zero or null then home screen is shown

### CODE REVIEW

File: lib/commands/general.js [hunk 1]
```diff
@@ -21,56 +21,42 @@ commands.active = async function active () {
...
-          params = {duration: duration.timeout / 1000.0};
-          endpoint = deactivateAppEndpoint;
-        } else {
-          endpoint = homescreenEndpoint;
-        }
+  let params = {};
+  const selectEndpoint = (timeoutSeconds) => {
+    if (!util.hasValue(timeoutSeconds)) {
+      endpoint = homescreen;
+    } if (!isNaN(timeoutSeconds)) {
```
Comment (imurchie) [inline lib/commands/general.js]: Not sure what this line is doing...

Comment (mykola-mokhnach) [reply lib/commands/general.js]: it check if timeoutSeconds is some random string or other object, which cannot be parsed to a number

Comment (imurchie) [reply lib/commands/general.js]: Why is it on that line?

Comment (mykola-mokhnach) [reply lib/commands/general.js]: where do you want to see it?

Comment (mykola-mokhnach) [reply lib/commands/general.js]: there was missing `else`. Should look better now

Comment (imurchie) [reply lib/commands/general.js]: Ah! That makes sense, then. Thanks.

**Pull request merged**